x-pack/plugins/security_solution/public/flyout/right/tabs/table_tab.tsx

```diff
@@ -5,16 +5,44 @@
  * 2.0.
  */
 
-import { EuiText } from '@elastic/eui';
 import type { FC } from 'react';
 import React, { memo } from 'react';
-import { TABLE_TAB_CONTENT_TEST_ID } from './test_ids';
+import { EuiEmptyPrompt } from '@elastic/eui';
+import { ERROR_MESSAGE, ERROR_TITLE } from './translations';
+import { TimelineTabs } from '../../../../common/types';
+import { EventFieldsBrowser } from '../../../common/components/event_details/event_fields_browser';
+import { useRightPanelContext } from '../context';
+import { TABLE_TAB_ERROR_TEST_ID } from './test_ids';
 
 /**
- * Table view displayed in the document details expandable flyout right section
+ * Table view displayed in the alert details expandable flyout right section
  */
 export const TableTab: FC = memo(() => {
-  return <EuiText data-test-subj={TABLE_TAB_CONTENT_TEST_ID}>{'Table tab'}</EuiText>;
+  const { browserFields, dataFormattedForFieldBrowser, eventId } = useRightPanelContext();
+
+  if (!browserFields || !eventId || !dataFormattedForFieldBrowser) {
+    return (
+      <EuiEmptyPrompt
+        iconType="error"
+        color="danger"
+        title={<h2>{ERROR_TITLE}</h2>}
+        body={<p>{ERROR_MESSAGE}</p>}
+        data-test-subj={TABLE_TAB_ERROR_TEST_ID}
+      />
+    );
+  }
+
+  return (
+    <EventFieldsBrowser
+      browserFields={browserFields}
+      data={dataFormattedForFieldBrowser}
+      eventId={eventId}
+      isDraggable={false}
```

Comment:
We need to determine what we do with this, as we'll have to pass some value for this for the flyout in the timeline context, though we'll probably want to take some time and look at the new timeline designs to figure out how we may want to treat this as well